contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -285,11 +285,14 @@ BEGIN
 		RAISE E'Could not initialize babelfish with given role name: % is not the DB owner of current database.', sa_name;
 	END IF;
 
+	EXECUTE format('CREATE ROLE securityadmin CREATEROLE INHERIT PASSWORD NULL');
+	EXECUTE format('GRANT CONNECT ON DATABASE %s TO securityadmin WITH GRANT OPTION', CURRENT_DATABASE());
```

Comment:
1. Why granting connect permission is required? 
2. Acc. ideal T-SQL behaviour, Does all the members of securityadmin can login to database always -- even if some login is disabled? Otherwise it doesn't make sense to add this permission.

Reply:
It's to grant connect permission on the babelfish physical database

Reply:
Why do we need connect to babelfish physical database permission ?

Reply:
We need this role to have atleast by default connect access on this babelfish physical database similar to our internal role, in fact for sysadmin, we provided create priv on this database which I think is necessarily not needed for our role we can just provide connect access to it.